src/lib.rs

```diff
@@ -20,29 +20,6 @@
 //!
 //! # Example
 //!
-//! ```rust
```

Comment:
why remove this?

Reply:
Because it fails for some reason with the new architecture, though all other tests pass. We'll have to write a new example doctest.

Reply:
👍 we should prob add a `todo` line

Reply:
An issue would probably be more effective. No one reads TODO lines. =)